guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-capsule-server.adoc

```diff
@@ -35,45 +35,41 @@ If you set a wildcard value `*` for the certificate's Common Name `CN =` in the
 If the command is successful, it returns two `{certs-generate}` commands, one of which you must use to generate the certificate archive file for your {SmartProxyServer}.
 ifdef::satellite[]
 +
-.Example output of `katello-certs-check`
+.To use them inside a _new-{smartproxy-example-com}_, run this command:
 [options="nowrap", subs="+quotes,attributes"]
 ----
-Validation succeeded.
-
-To use them inside a NEW $CAPSULE, run this command:
-
 capsule-certs-generate --foreman-proxy-fqdn "$CAPSULE" \
     --certs-tar  "~/$CAPSULE-certs.tar" \
     --server-cert "_/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem_" \
     --server-key "_/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem_" \
     --server-ca-cert "_/root/{smart-proxy-context}_cert/ca_cert_bundle.pem_" \
-
-To use them inside an EXISTING $CAPSULE, run this command INSTEAD:
-
+----
++
+.To use them inside an _existing-{smartproxy-example-com}_, run this command INSTEAD:
```

Comment:
I know it was there before, but I wonder why `INSTEAD` is in caps, but I'm not sure it makes that much sense. Before it was there to highlight the difference to a new, but you removed those. I'm worried the hostname may be too subtle for users to distinguish.

Reply:
I understand the inconsistencies in this version of the guide. I have found a few more things and modified them.